### PR TITLE
Update 5.2_Find_Usages.php

### PR DESCRIPTION
**Pull request by tugrulcan**
This PR updates the `Show Usages` pop-up shortcut and adds a note about a system hotkey in Ubuntu

### CODE REVIEW

File: workshop/01_Navigation/05_Find_usages/5.2_Find_Usages.php [hunk 1]
```diff
@@ -6,10 +6,13 @@
  *
  * Show Usages (uses popup)
  *
- * Ctrl+Alt+F7 (Windows/Linux)
+ * Ctrl+Alt+7 (Windows/Linux)
```
Comment (MaXal) [inline workshop/01_Navigation/05_Find_usages/5.2_Find_Usages.php]: Please leave the default shortcut as it is.

File: workshop/01_Navigation/05_Find_usages/5.2_Find_Usages.php [hunk 2]
```diff
@@ -6,10 +6,13 @@
...
  * Alt+Command+F7 (macOS)
  *
  * Highlight usages of a given symbol in the current project.
+ *
+ * NOTE: On Ubuntu 18.04 and newer, Alt+F7 is overwritten
```
Comment (MaXal) [inline workshop/01_Navigation/05_Find_usages/5.2_Find_Usages.php]: Thank you! This information is indeed useful!

File: workshop/01_Navigation/05_Find_usages/5.2_Find_Usages.php (hunk 1 above)
Comment (tugrulcan) [reply workshop/01_Navigation/05_Find_usages/5.2_Find_Usages.php]: This was the default shortcut for Ubuntu.

File: workshop/01_Navigation/05_Find_usages/5.2_Find_Usages.php (hunk 2 above)
Comment (tugrulcan) [reply workshop/01_Navigation/05_Find_usages/5.2_Find_Usages.php]: Thanks for pointing that out :+1:

**Comment by tugrulcan** [comment]
Hi @MaXal 
I have updated the comment

**Pull request merged**